sdk/tools/pom.xml

```diff
@@ -13,5 +13,6 @@
     <module>azure-sdk-build-tool</module>
     <module>azure-openrewrite</module>
     <module>azure-openrewrite-compiler-maven-plugin</module>
+    <module>sdk-linting-extensions</module>
```

Comment:
Should we use `eng/tools` directory for this? Generally, the `sdk/tools` directory is used for tools that are meant to be used by customers. These linting extensions are mainly for library developers.